website/showcase.json

```
@@ -214,7 +214,7 @@
   },
   {
     "name": "Artsy",
-    "icon": "https://raw.githubusercontent.com/artsy/eigen/master/Artsy/Resources/Images.xcassets/AppIcon.appiconset/AppIcon167.png",
+    "icon": "https://avatars.githubusercontent.com/u/546231?s=200&v=4",
```

```suggestion
    "icon": "artsy.png",
```
any reasons why you can't change it to use the .png icon that is already in the repo?

---

i changes it because previous url of image path was not working. 
i didn't checked artsy.png in repo so i added that link to github avatars

---

yes